src/platform/web/ui/general/StaticView.ts

```diff
@@ -15,15 +15,17 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import {tag} from "../general/html";
+import {tag} from "./html";
 
 export class StaticView {
```

Comment:
```suggestion
export class StaticView<T> {
```

With T being the type of the view model used.